connectors/es/client.py

```diff
@@ -101,9 +103,11 @@ def __init__(self, config):
         self.max_wait_duration = config.get("max_wait_duration", 60)
         self.initial_backoff_duration = config.get("initial_backoff_duration", 5)
         self.backoff_multiplier = config.get("backoff_multiplier", 2)
+
         options["headers"] = config.get("headers", {})
-        options["headers"]["user-agent"] = f"elastic-connectors-{__version__}"
+        options["headers"]["user-agent"] = self.__class__.user_agent
```

Comment:
Should we establish a "base" with both usages providing a suffix? Or should only the CLI have a suffix?
Like should the normal agent be changed to have a suffix like: 
```
        options["headers"]["user-agent"] = f"{self.__class__.user_agent}/service"
```

Reply:
I like having `/service` as an explicit suffix, does not leave room for interpretation 👍